x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_response_actions_console_commands.ts

```diff
@@ -15,11 +15,26 @@ import { EndpointStatusActionResult } from './status_action';
 import { GetProcessesActionResult } from './get_processes_action';
 import type { ParsedArgData } from '../console/service/parsed_command_input';
 
-const emptyArgumentValidator = (argData: ParsedArgData) => {
+const emptyArgumentValidator = (argData: ParsedArgData): true | string => {
   if (argData?.length > 0 && argData[0]?.trim().length > 0) {
     return true;
   } else {
-    return 'Argument cannot be empty';
+    return i18n.translate('xpack.securitySolution.endpointConsoleCommands.emptyArgumentMessage', {
+      defaultMessage: 'Argument cannot be empty',
+    });
+  }
+};
+
+const pidValidator = (argData: ParsedArgData): true | string => {
+  const emptyResult = emptyArgumentValidator(argData);
+  if (emptyResult !== true) {
+    return emptyResult;
+  } else if (Number.isInteger(Number(argData)) && Number(argData) > 0) {
```

Comment:
`Number.isInteger(Number(argData))` is not necessary here. `Number(argData) > 0` is enough to correctly check if the argument is a positive number